tensorflow/contrib/kernel_methods/python/kernel_estimators_test.py

```diff
@@ -92,7 +92,7 @@ def testLinearlySeparableBinaryDataNoKernels(self):
     # Since the data is linearly separable, the classifier should have small
     # loss and perfect accuracy.
     self.assertLess(metrics['loss'], 0.1)
-    self.assertEqual(metrics['accuracy'], 1.0)
+    self.assertAlmostEqual(metrics['accuracy'], 1.0, 7)
```

Comment:
The reason for that was the fact that sometimes OpenCL represent this value as 0.99999996
Reverting in 1a6045241a27228ba817cb0b722679c357077e36